jupyter_server/serverapp.py

```diff
@@ -541,6 +542,17 @@ def start(self):
     'gateway-url': 'GatewayClient.url',
 })
 
+classes = [
+        KernelManager, Session, MappingKernelManager, KernelSpecManager,
+        ContentsManager, FileContentsManager, NotebookNotary,
+        GatewayKernelManager, GatewayKernelSpecManager, GatewaySessionManager, GatewayClient
+    ]
+try:
+    from .services.kernels.kernelmanager import AsyncMappingKernelManager
+    classes.append(AsyncMappingKernelManager)
+except ImportError:
+    pass
```

Comment:
Same as previous: shouldn't be necessary given we're pinning the floor of jupyter_client.